impeller/entity/shaders/filters/gaussian.frag

```diff
@@ -11,14 +11,11 @@ uniform f16sampler2D texture_sampler;
 
 layout(constant_id = 0) const float supports_decal = 1.0;
 
-struct KernelSample {
-  vec2 uv_offset;
-  float coefficient;
-};
-
-uniform KernelSamples {
-  int sample_count;
-  KernelSample samples[50];
+uniform BlurInfo {
```

Comment:
I think the name `KernelSamples` is more apropos.

Reply:
Fun fact, the uniform struct name has to exactly match the uniform declaration name below for the GLES backend 😆

Reply:
Wait, what!?  "BlurInfo" has to match "blur_info"?  How does it know, it's converting camel case to snake case?

Reply:
Its the magic UBO destructuring in the GLES backend 😈

Reply:
SOYLENT GREEN IS ~PEOPLE~ STRINGS

Reply:
Changed to KernelSamples